app/models/comment.rb

```diff
@@ -1,9 +1,16 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User'
   belongs_to :post, class_name: 'Post'
-  after_save :update_comments_counter
+  after_save :update_posts_counter
 
-  def update_comments_counter
+  def update_posts_counter
     post.increment!(:comments_counter)
   end
 end
+
+# Test data...I will delete after prject input fields are created
+# Comment.create(post: fifth_post, author: fifth_user, text: 'Hi Sam!' )
+# Comment.create(post: fifth_post, author: third_user, text: 'Hi Sam!' )
+# Comment.create(post: fourth_post, author: first_user, text: 'Hi Dan!' )
+# Comment.create(post: sixth_post, author: fourth_user, text: 'Hi Sam!' )
```

Comment:
- [ ] Please remove the commented code from your code to improve its quality and have clean code

Reply:
I was hoping you don't ask for me to remove it 😄. I tried explaining why I left it but I guess I can just put them in an external file and add the file to .gitignore